packages/engine/src/worker/mod.rs

```diff
@@ -418,7 +416,7 @@ impl WorkerController {
         source: Language,
     ) -> Result<()> {
         if let Some(pending) = self.tasks.inner.get_mut(&msg.task_id) {
-            // TODO: implement worker node handler
+            // TODO: `handle_worker_message` always returns `Error::WorkerNodeHandlerNotImplemented`
```

Comment:
I don't think this is a TODO. I don't believe any packages currently have tasks that have dynamic targets so we wouldn't have needed to implement this yet. 

@litvand, do you reckon I'm right on that?

Reply:
I think so. Returning "NotImplemented" is just a placeholder implementation.

Reply:
Resolving, has been removed